index.js

```diff
@@ -231,16 +239,25 @@ const init = async () => {
     });
   });
 
-  // Then we load events, which will include our message and ready event.
-  const evtFiles = await readdir("./events/");
-  evtFiles.forEach(file => {
-    const eventName = file.split(".")[0];
-    client.logger.log(`Loading Event: ${eventName}. 👌`, "log");
```

Comment:
Nooo, my boy... you removed this lol

Reply:
Well, yes, as it was replaced by a new event loader. Seems kinda silly to have both.

Reply:
This was more of a meme comment, but from a personal point of view I would rather have a list of events that successfully loaded over a blanket statement of X events loaded, like it originally was, and like how the commands, and slash commands are.

Reply:
Changed back in latest commit.